cmd/juju/status/history.go

```diff
@@ -47,13 +48,13 @@ This command will report the history of status changes for
 a given entity.
 The statuses are available for the following types.
 -type supports:
-    juju-unit: will show statuses for the unit's juju agent.
-    workload: will show statuses for the unit's workload.
-    unit: will show workload and juju agent combined for the specified unit.
+    container: will show statuses for containers.
+    juju-container: will show statuses for the container's juju agent.
     juju-machine: will show statuses for machine's juju agent.
+    juju-unit: will show statuses for the unit's juju agent.
     machine: will show statuses for machines.
-    juju-container: will show statuses for the container's juju agent.
-    container: will show statuses for containers.
+    unit: will show workload and juju agent combined for the specified unit.
+    workload: will show statuses for the unit's workload.
  and sorted by time of occurrence.
```

Comment:
What if we just added descriptions to something like AllHistoryKind and then built this list from that? Then all the updating would be in the same place.

A different option would be to check the help text with something like:
def TestHasAllKinds(...) {
  for kind in AllHistoryKinds() {
    c.Assert(fmt.Sprintf("    %s: ", kind), IsIn, docstring)

What that very much *doesn't* get you is the negative that no other types are in the documentation, which aren't actually supported.

A different way would be to keep a local map of type name to doc and then just a test that the keys in the map match the AllHistoryKinds.

Something like:
typeDocs := map[string]string {
  "container": "will show statuses for containers",
  "juju-container": "will show statuses for the container's juju agent"
...
}

func TestHistoryKinds()  {
   all := set.NewStrings(AllHistoryKind()...)
   for kind, _ := range typeDocs {
     c.Assert(all.Contains(kind), jc.IsTrue)
     all.Remove(kind)
   }
   c.Assert(all.SortedValues(), jc.DeepEquals, []string{})

Thoughts?